### PR TITLE
Changes in response to JOSS review

### PR DESCRIPTION
**Pull request by mommermi**
This PR addresses the following issues in response to the JOSS review process (openjournals/joss-reviews#1426):

NASA-Planetary-Science/sbpy#149
NASA-Planetary-Science/sbpy#150
NASA-Planetary-Science/sbpy#152
NASA-Planetary-Science/sbpy#153
NASA-Planetary-Science/sbpy#154
NASA-Planetary-Science/sbpy#155

NASA-Planetary-Science/sbpy#151 has already been closed.

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -33,21 +33,50 @@ that are stable and which you can already use for your research. We
...
+  <http://docs.astropy.org/en/latest/development/docguide.html>`__.
+  Code should be written following the `astropy coding guidelines
+  <http://docs.astropy.org/en/latest/development/codeguide.html>`__.
+* `sbpy` follows the `astropy testing guidelines
+  <http://docs.astropy.org/en/latest/development/testguide.html>`__.
+* Once you are done working on your code, push your branch to your
+  forked repository on github and create a pull request describing
+  your code and linking to your original issue report.
+  
+Please follow the `astropy code of conduct`_ at any time.
```
Comment (bsipocz) [inline docs/contributing.rst]: If you also add a `CONTRUBUTING.md` and `CODE_OF_CONDUCT.md` to the root of the repo they'll come up as links for people on issues/PRs. Alternatively they can be added to a central `.github` repo and then they'll show up for all repos in that org.

File: sbpy/data/ephem.py [hunk 2]
```diff
@@ -62,6 +62,16 @@ def from_horizons(cls, targetids, id_type='smallbody',
...
 
+        Notes
+        -----
+        * For detailed explanations of the queried fields, refer to
+          `astroquery.jplhorizons.HorizonsClass.ephemerides <https://astroquery.readthedocs.io/en/latest/api/astroquery.jplhorizons.HorizonsClass.html#astroquery.jplhorizons.HorizonsClass.ephemerides>`_ and the
+          `JPL Horizons documentation <https://ssd.jpl.nasa.gov/?horizons_doc>`_.
+        * By default, all properties are provided in the J2000.0 reference
+          system. Different settings can be chosen using
+          additional keyword arguments as used by
+          `astroquery.jplhorizons.HorizonsClass.ephemerides <https://astroquery.readthedocs.io/en/latest/api/astroquery.jplhorizons.HorizonsClass.html#astroquery.jplhorizons.HorizonsClass.ephemerides>`_.
```
Comment (bsipocz) [inline sbpy/data/ephem.py]: I would suggest to use intersphinx instead to link to API docs in other packages as the actual links may change even when the API is kept the same.

**Comment by bsipocz** [comment]
I have only minor suggestions, neither of them are required at all for this PR.

File: docs/contributing.rst (hunk 1 above)
Comment (mommermi) [reply docs/contributing.rst]: That's a great idea, thanks! Done!

File: sbpy/data/ephem.py (hunk 2 above)
Comment (mommermi) [reply sbpy/data/ephem.py]: You can read my mind! Intersphinx is exactly what I was looking for! Thanks!

File: docs/sbpy/data.rst [hunk 3]
```diff
@@ -19,6 +19,73 @@ different services and `~sbpy.data.Names` provides functions
...
+`~sbpy.data.Phys.from_sbdb`.
+
+
+Names
+^^^^^
+
+`~sbpy.data.Names` objects are somewhat different from the other data
+containers, as they don't hold properties but only object
+**names**. These names can be used to identify object nature
+(`~sbpy.data.Names.asteroi_or_comet`) and they can be parsed to
```
Comment (Juanlu001) [inline docs/sbpy/data.rst]:
```suggestion
(`~sbpy.data.Names.asteroid_or_comet`) and they can be parsed to
```

Comment (mommermi) [reply docs/sbpy/data.rst]: Done, thanks!

**Comment by bsipocz** [comment]
@mommermi - no need to wait with merging this, master is broken as well if you were to restart the latest build. The astropy issue is merely a packaging issue, I don't know when it will be resolved though.

**Pull request merged**